_deploy/r/demo/gns_halving/gns.gno

```diff
@@ -120,3 +120,12 @@ func Mint(address pusers.AddressOrName) {
 		// println("height:", i, "minted:", amount)
 	}
 }
+
```

Comment:
XXX: Remove this

Reply:
92eedc4d4a3245e6a9376a75bac78d54e9a5f389